### DISCUSSION TITLE
Best Implementation of Asymmetrical Photo

### DISCUSSION THREAD

**Discussion by kmarler18** [tailwindlabs/tailwindcss#10195] [Help]
Hey ya'll! 

I'm trying to implement the following design in my website.

![Screen Shot 2022-12-31 at 1 00 37 AM](https://user-images.githubusercontent.com/96091818/210126891-b171a958-2613-4113-88c1-755f759263e7.png)

I've been given two .png files pictured below.

![Screen Shot 2022-12-31 at 1 04 36 AM](https://user-images.githubusercontent.com/96091818/210126932-b745c31a-244c-4ba8-8d52-8005bd5d3e58.png)
![Screen Shot 2022-12-31 at 1 04 48 AM](https://user-images.githubusercontent.com/96091818/210126933-156a4a7b-0174-4835-a3ed-eac7c1cd9f24.png)

What do you think is the most effective solution to implement this design?

Happy last day of 2022,
Kyle

**Comment by wongjn** [comment]
Have these two images displayed on stacked on top of each other in the z-axis. Edit the first image such that it is the desired heart shape or use [`clip-path`](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path).

**Reply by kmarler18** [reply]
This ended up getting resolved in [this thread](https://github.com/tailwindlabs/tailwindcss/discussions/10197).
